packages/utilities/gel-foundations/src/breakpoints.js

```diff
@@ -3,6 +3,7 @@
    These namings are based on the GEL description. They are also known as group B and group D
```

Comment:
Could you please increase the version number in the `package.json` and `package-lock.json` & also add a line in the `CHANGELOG.md`?